MapboxNavigationTests/ManeuverViewTests.swift

```diff
@@ -69,3 +88,13 @@ class ManeuverViewTests: FBSnapshotTestCase {
     // TODO: Figure out why the flip transformation do not render in a snapshot so we can test left turns and left side rule of the road.
 }
 
+extension UIView {
+    
+    func asImage() -> UIImage {
```

Comment:
Consider replacing this method with an initializer in a UIImage extension. That style is less “fluent” but also more consistent with the majority of the standard library.

Reply:
I agree but initializers on UIImage would require more steps and it's not particularly elegant since you can't return the instance. It would involve UIView->UIImage->CGImage(or any other type supported by UIImage->UIImage. So I went with a class function on UIImage instead, `UIImage.from(_:)`.